skopt/optimizer/optimizer.py

```diff
@@ -528,3 +460,95 @@ def run(self, func, n_iter=1):
 
         return create_result(self.Xi, self.yi, self.space, self.rng,
                              models=self.models)
+
+    def fit(self, mask=None):
```

Comment:
should we make this private? `_fit()` it isn't meant to be called by anyone right?

Reply:
Wouldn't it make more sense to have `ask` call `fit`? Then we can make `mask` a parameter of `ask`.

Reply:
Not sure I understand your comment :-/ `ask` should indeed call `fit` but I think it should be a private method, not something users of `Optimizer` have access to.

Reply:
Take a look at latest commit, and let me know what you think.